pkg/apis/vpc/v1alpha1/subnetconnectionbindingmap_types.go

```diff
@@ -34,6 +37,7 @@ type SubnetConnectionBindingMapStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:storageversion
 // +kubebuilder:resource:scope="Namespaced",path=subnetconnectionbindingmaps,shortName=subnetbinding;subnetbindings
+// +kubebuilder:selectablefield:JSONPath=`.spec.subnetName`
```

Comment:
what's the purpose of this change?

Reply:
We want to check if there are any existing BindingMap CRs still reference the shared Subnet, refuse to unassign the Subnet from Namespace if the assigned Subnet is still used.

Reply:
Got it.